### PR TITLE
Remove label of cases that have been solved

### PR DESCRIPTION
**Pull request by NicoYuan1986**
Issue: #18908
Signed-off-by: “nico” <Nico_1986@163.com>

### CODE REVIEW

File: tests/python_client/testcases/test_query.py [hunk 1]
```diff
@@ -269,6 +269,7 @@ def test_query_expr_non_primary_fields(self):
                                        check_task=CheckTasks.check_query_results, check_items={exp_res: res})
 
     @pytest.mark.tags(CaseLabel.L2)
+    # @pytest.mark.xfail(reason="issue #12210 #7522")
```
Comment (binbinlv) [inline tests/python_client/testcases/test_query.py:272]: This could be deleted.

Comment (NicoYuan1986) [reply tests/python_client/testcases/test_query.py]: > This could be deleted.

Done!

**Pull request merged**